mongorm/fields/ReferenceField.py

```diff
@@ -9,7 +9,7 @@
 class ReferenceField(BaseField):
 	def __init__( self, documentClass, *args, **kwargs ):
 		super(ReferenceField, self).__init__( *args, **kwargs )
-		
+		self._use_ref_id = kwargs.get('use_ref_id', False)
```

Comment:
How does this get passed in? Does it work for recursive dereferencing of ReferenceFields?

Reply:
This would be passed in the Document field declaration as a kwarg eg ReferenceField(UserAccount, use_ref_id=True)